eternalcore-plugin/build.gradle.kts

```diff
@@ -34,6 +34,6 @@ eternalShadowCompiler {
 
 tasks {
     runServer {
-        minecraftVersion("1.20")
+        minecraftVersion("1.20.2")
```

Comment:
```suggestion
        minecraftVersion("1.20.4")
```

Reply:
Paper 1.20.4 is experimental and unstable, currently I want to stay with 1.20.2 until the stable version is released.